task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -266,37 +276,35 @@ def subprocess_main():
             [
                 {
                     "chan": "stdout",
-                    "event": "I'm a short message",
+                    "event": "supervisor.py: I'm a short message",
```

Comment:
This defeats the purpose of structlog.
```suggestion
                    "event": "I'm a short message",
                    "filename": "supervisor.py"
```

etc